bcgov/mds
# MDS-4058: View additional project data in Core

# Main

- https://bcmines.atlassian.net/jira/software/c/projects/MDS/boards/1?modal=detail&selectedIssue=MDS-4058

# How to test

- Visit Core -> Mines -> Select a mine with project descriptions ->Permits & Approvals - Pre-applications - > Open on a project description
- Verify side menu navigation works and that all information entered from the Minespace side shows up
